kani-compiler/src/codegen_cprover_gotoc/context/goto_ctx.rs

```diff
@@ -39,6 +39,7 @@ use stable_mir::mir::mono::Instance;
 use stable_mir::mir::Body;
 use stable_mir::ty::Allocation;
 
+#[allow(dead_code)]
```

Comment:
Why was this added? It might be better to address the cause of the warning (e.g. delete unused code) rather than suppress the warning.

Reply:
Removed all dead code as suggested.